tests/test_default_values.py

```diff
@@ -0,0 +1,108 @@
+"""Copyright 2019 Equinor ASA and The Netherlands Organisation for
+Applied Scientific Research TNO.
+
+Licensed under the MIT license.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the conditions stated in the LICENSE file in the project root for
+details.
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+"""
+
+import unittest
+
+import configsuite
+from configsuite import MetaKeys as MK
+from configsuite import types
+from .data import car
+
+
+class TestDefaultValues(unittest.TestCase):
+    def test_default_values(self):
+        raw_config = car.build_all_default_config()
+        car_schema = car.build_schema()
+        config_suite = configsuite.ConfigSuite(raw_config, car_schema)
+
+        self.assertTrue(config_suite.valid)
+
+        # Unspecified containers are defaulted to empty container of their type
```

Comment:
Please verify that `config_suite.tire.rim` and `config_suite.tire.color` is defaulted as expected.